hooks/pre-command

```diff
@@ -26,6 +26,12 @@ done < <(env | sort)
 
 
 if [ "${#paths[@]}" -gt 0 ]; then
-  echo "--- Downloading artifacts"
-  buildkite-agent artifact download "$(IFS=, ; echo "${paths[*]}")" "${BUILDKITE_PLUGIN_ARTIFACTS_WORKDIR}"
+  build=${BUILDKITE_PLUGIN_ARTIFACTS_BUILD:-}
```

Comment:
Generally the best way to handle this conditional arguments stuff is with an `args` array, like in https://github.com/buildkite-plugins/docker-buildkite-plugin/blob/master/hooks/command#L12